BLSQ/iaso
# IA-2859 add payment perm to front end

Payment permission is not accessible to non super users

Related JIRA tickets : IA-2859

## Self proofreading checklist


## Doc
Added `/docs/pages/dev/how_to/add_new_permission`

## Changes

- Add Payments module in the backend (permission itself was already there), with the payments permission
- Add missing translations for the permission and the module

## How to test

- Go to Modules page: permission module should be visible
- Go to django admin and add the module to your current account
- Go to users page: the payments permission should be visible in the tab
- Give the permission to non-super user then log as this user: the payments page should be available in the menu

## Print screen / video

![Screenshot 2024-04-10 at 12 25 34](https://github.com/BLSQ/iaso/assets/38907762/4aa64bcc-9a48-404c-b91f-c006bdb73fbf)

![Screenshot 2024-04-10 at 12 24 54](https://github.com/BLSQ/iaso/assets/38907762/02637c00-ab5e-4f31-b7e5-a85cbe165968)

![Screenshot 2024-04-10 at 12 25 19](https://github.com/BLSQ/iaso/assets/38907762/4dd63715-5e87-4f40-99a3-4ffc7ea6ba20)
